kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -615,9 +617,10 @@
     padding-right: 16px;
   }
 
-  .hint-btn-container {
+  /deep/ .hint-btn-container {
```

Comment:
Is this class still relevant any more? We've moved it out of the perseus renderer, so this might just be left over from my failing to clean it up?

Reply:
Ah yep I copied this over but ought to have cut, will remove shortly